### ISSUE TITLE
DualBoot with Win 10 disabled after the latest Feren OS updates

### ISSUE THREAD

**Issue by pcdr1** [feren-OS/Bug-Reporting-Center#199]
I am currently running latest Feren OS based on KDE 5.20.4. Everything was working beautifully until the latest incremental updates Feren released on 15 Jan 2021. I am unable to select the option to boot into Windows 10 on initial Grub Menu. Any Help?
![Screenshot_20210116_192114](https://user-images.githubusercontent.com/77560685/104830622-3de71f00-5835-11eb-9350-8c5ffd7e2ad4.png)

**Comment by dominichayesferen** [comment]
Try running this in Terminal:
`sudo update-grub`

sudo password = your password

**Comment by pcdr1** [comment]
Did that via terminal..No Help.Tried to do boot-repair using the bootable usb...still no help.
Basically the boot menu is there but cannot do the selection as previous to the update to move the cursor. 
Tried to plug in an external usb keyboard..Same results. Unable to move cursor to select Advanced Option or Windows 10.
Thanks for any assistance, Bill


Sent from Yahoo Mail on Android 
 
  On Tue, Jan 19, 2021 at 22:27, Dominic Hayes - Feren OS Dev<notifications@github.com> wrote:   


Try running this in Terminal:
sudo update-grub

sudo password = your password

**Comment by dominichayesferen** [comment]
Weird... are you on modern hardware, by chance? (I guess the easiest way to check would be to see if there's a folder at File System/boot/efi called 'EFI')

**Comment by pcdr1** [comment]
hi, there is no such folder as you can see on the enclosed screenshot. System is an old HP Elitebook 8760w running on 8 gig of ram. been using Feren OS for more than 2 years. First real issue that I cannot figure out what is going on. I rolled it back via Timeshift 7 days ago but apparently problem started before that. I am considering reinstalling Feren OS using the NOV release and stop the auto update as a last resort if we cannot get this resolved. thanks in advance for any assistance. Bill
    On Wednesday, January 20, 2021, 03:15:25 PM PST, Dominic Hayes - Feren OS Dev <notifications@github.com> wrote:  
 
 


Weird... are you on modern hardware, by chance? (I guess the easiest way to check would be to see if there's a folder at File System/boot/efi called 'EFI')

**Comment by dominichayesferen** [comment]
_FYI your 'enclosed' screenshot didn't send._

Very odd... it's probably be good to try the January 2021 Snapshot as a reinstall, as over on my machine with Windows 7 it still allows booting into Windows 7 via GRUB.

(there again I suppose it's better that this happens than Windows 10 deciding to mess up GRUB itself as it usually does)

**Comment by pcdr1** [comment]
It is really whierd indeed. I tried the grub menu selection again after I grabbed the latest update via the update manager as usual. Everything worked normally again as it should. All issues resolved by itself as I previously reported. I did not have to reload the Feren OS from scratch after all. I believe it was something caused the issue on Linux updates between the 7th to the 14th of Jan since my roll back via timeshift on the 14th didn't clear up the problem.All is well now but cause is unknown other than the reported symptoms. It resolved itself on this latest update I grabbed on or about the 22nd. Thanks Again, Bill

Sent from Yahoo Mail on Android 
 
  On Sun, Jan 24, 2021 at 1:22, Dominic Hayes - Feren OS Dev<notifications@github.com> wrote:   


FYI your 'enclosed' screenshot didn't send.

Very odd... it's probably be good to try the January 2021 Snapshot as a reinstall, as over on my machine with Windows 7 it still allows booting into Windows 7 via GRUB.

(there again I suppose it's better that this happens than Windows 10 deciding to mess up GRUB itself as it usually does)
